daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -721,6 +721,32 @@ class RazerHuntsmanElite(_RippleKeyboard):
     }
 
 
+class RazerHuntsman(_RippleKeyboard):
+    """
+    Class for the Razer Huntsman
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*Huntsman(-if01)?-event-kbd')
+
+    USB_VID = 0x1532
+    USB_PID = 0x0227
+    HAS_MATRIX = True
+    WAVE_DIRS = (0, 1)
+    MATRIX_DIMS = [6, 22]
+    METHODS = ['get_device_type_keyboard', 'set_wave_effect', 'set_static_effect', 'set_spectrum_effect',
+               'set_reactive_effect', 'set_none_effect', 'set_breath_random_effect', 'set_breath_single_effect', 'set_breath_dual_effect',
+               'set_custom_effect', 'set_key_row', 'get_game_mode', 'set_game_mode', 'get_macro_mode', 'set_macro_mode',
+               'get_macro_effect', 'set_macro_effect', 'get_macros', 'delete_macro', 'add_macro',
+               'set_starlight_random_effect', 'set_starlight_single_effect', 'set_starlight_dual_effect',
+               'set_ripple_effect', 'set_ripple_effect_random_colour']
+
+    DEVICE_IMAGE = "https://d4kkpd69xt9l7.cloudfront.net/sys-master/root/h03/hd8/9016073224222/razer-huntsman-gallery01.jpg"
```

Comment:
Please use https://assets.razerzone.com/eeimages/support/products/1360/1360_huntsman-3.png for this and the url below